app/services/client_login_service.rb

```diff
@@ -28,6 +28,10 @@ def can_login_by_email_verification?(email_address)
   end
 
   def can_login_by_sms_verification?(sms_phone_number)
-    service_class.accessible_intakes.where(sms_phone_number: sms_phone_number, sms_notification_opt_in: "yes").exists?
+    if service_class == Intake::CtcIntake || service_class == Intake::GyrIntake
+      return service_class.accessible_intakes.where(sms_phone_number: sms_phone_number, sms_notification_opt_in: "yes").exists?
+    elsif service_class == StateFileAzIntake || service_class == StateFileNyIntake
+      return service_class.accessible_intakes.where(phone_number: sms_phone_number).exists?
```

Comment:
I think it would make sense to use `can_be_authenticated` here in place of `accessible_intakes`